logs/querysample/tags.go

```diff
@@ -53,6 +53,17 @@ func parseTags(query string) map[string]string {
 	return tags
 }
 
+func sqlcommenterFormat(str string) bool {
+	keyAndValue := strings.SplitN(str, "=", 2)
+	if len(keyAndValue) == 2 {
+		// With sqlcommenter format (key='value'), key shouldn't include ":"
+		if !strings.Contains(keyAndValue[0], ":") {
+			return true
+		}
+	}
+	return false
```

Comment:
NBD, but this could be simplified to
```suggestion
        // With sqlcommenter format (key='value'), key shouldn't include ":"
        return len(keyAndValue) == 2 && !strings.Contains(keyAndValue[0], ":")
```